### PR TITLE
Auto-classify IBHE-reported units (i.e., higher ed institutions)

### PR DESCRIPTION
**Pull request by hancush**
This PR:

- Adds auto-classification of units reported by the IBHE.
- Repairs a longstanding (but non-fatal) issue with the indexing task, and adds a small batchsize so entities appear on the site as they are indexed, rather than all at once.
- Updates the tests to reflect the change in import functionality.

### CODE REVIEW

**Comment by hancush** [comment]
This issue ultimately occurred because there was a mismatch between athletic department data and the FOIA source lookup that associated higher ed units with the ISBE responding agency. I am currently waiting on an updated [FOIA source lookup table](https://github.com/datamade/bga-payroll/blob/master/data/raw/foia-source-lookup.csv) from BGA. With that in hand, the remade athletic department data will satisfy the condition in [the new import query](https://github.com/datamade/bga-payroll/pull/323/files#diff-5515aa86b46d2da9cbf6311bb937c0b2R217) and be correctly (and automatically) classified on import.

File: data/foia_source.mk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+data/output/foia-source-lookup.csv : data/raw/foia-source-lookup.csv data/raw/foia-source-lookup.full.csv
+	csvstack $^ | \
+	csvsql --query 'SELECT MIN(Employer), ID FROM STDIN GROUP BY ID' | \
```
Comment (fgregg) [inline data/foia_source.mk]: this is a little concerning. are there records with the same ID but different names and how sure are we that that is safe.

Comment (hancush) [reply data/foia_source.mk]: thank you for this catch. i added a query to spit out instances where employer differed in more than casing for one id. many of the instances look like the same thing, however [there are some that are totally different](https://gist.github.com/hancush/a27174053859410ff71b99d95726bf06#file-duplicate-audit-csv-L18-L25). i think we need more information about this lookup (and how they assign the ids on their end) before we push this broader change forward. can we chat about an interim step to get the ad data out the door?

Comment (fgregg) [reply data/foia_source.mk]: sure. Call me at 1:45?

Comment (hancush) [reply data/foia_source.mk]: will do, thanks!

Comment (hancush) [reply data/foia_source.mk]: verdict: i am going to undo lookup-related changes and add ibhe to the original lookup file, which will resolve our immediate problem. for the future, i’ll open an issue to acquire a complete, canonical lookup table from bga.

Comment (hancush) [reply data/foia_source.mk]: opened #325.

**Pull request merged**